constantine-rust/constantine-halo2-zal/Cargo.toml

```diff
@@ -22,7 +23,7 @@ rand_xorshift = "0.3"
 rayon = "1.8"
 
 # In CI "asm" needs to be disabled as some agents don't support ADX.
-halo2curves = { git = 'https://github.com/taikoxyz/halo2curves', branch = "pr-pse-exec-engine" }
+# halo2curves = { git = 'https://github.com/taikoxyz/halo2curves', branch = "pr-pse-exec-engine" }
```

Comment:
Not sure what should happen with this here.

Reply:
Leftover from when I enabled multicore and asm features for benching (and disabled it in CI) See: [231ccc7#diff-86e33129ff3e1c7ef4e097a64b3bba9f931352847b95fa0d7c9033665c10b8b6R26](https://github.com/mratsim/constantine/commit/231ccc7acea7975267a49e88e50035b137539b61#diff-86e33129ff3e1c7ef4e097a64b3bba9f931352847b95fa0d7c9033665c10b8b6R26)

It can be deleted with.

Note that if upstream is changed to reexport part of halo2curves, we might need it again for the benchmarks:

https://github.com/mratsim/constantine/blob/12f568654e7b6d7b9b738c4bd3d615932bde8d47/constantine-rust/constantine-halo2-zal/benches/msm.rs#L18-L21

at least the best_multi_exp part.